docs/extensions/fares-v2.md

```diff
@@ -0,0 +1,82 @@
+<a class="pencil-link" href="https://github.com/MobilityData/gtfs.org/edit/main/docs/extensions/fare-extension.md" title="Edit this page" target="_blank">
+    <svg class="pencil" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M10 20H6V4h7v5h5v3.1l2-2V8l-6-6H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h4v-2m10.2-7c.1 0 .3.1.4.2l1.3 1.3c.2.2.2.6 0 .8l-1 1-2.1-2.1 1-1c.1-.1.2-.2.4-.2m0 3.9L14.1 23H12v-2.1l6.1-6.1 2.1 2.1Z"></path></svg>
+  </a>
+  
+# GTFS-Fares v2
+
+Fares v2 is a GTFS Schedule extension project that aims to address the limitations of <a href="/schedule/examples/fares-v1">Fares v1</a>.
+
+The main concepts that Fares v2 plans to represent are
+
+- Fare products (e.g tickets and passes)
+- Rider categories (e.g seniors and children)
+- Fare payment options (e.g bank card and credit card)
+- Fare capping
+
+These concepts will allow data producers to model zone-based, time-dependent, and inter-agency fares. This extension project is being adopted in iterations. 
+
+Currently, the officially adopted base implementation of Fares v2 works for the following use cases:
+
+- Defining a transit fare
+- Describing service locations in the same fare zone
+- Creating rules for single leg journeys
+- Creating rules for transfers
+
+You can see <a href="/schedule/examples/fares-v2" target="_blank">examples here</a> that show what can be modelled using the official Fares v2 base implementation.
```

Comment:
I think "official" is not really defined. We could argue that the [Google Doc](https://docs.google.com/document/d/19j-f-wZ5C_kYXmkLBye1g42U-kvfSVgYLkkG5oyBauY/edit) is the official GTFS Fares v2. 
I think using "adopted" might be clearer.